### ISSUE TITLE
NTR: Lice and/or nits in hair or eyelashes (Hair)	

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7999]
NTR: Lice and/or nits in hair or eyelashes (Hair)	New term request Lice and/or nits in hair or eyelashes (seidel_190324162046): Pediculosis capitis; head lice; louse eggs; nymphs; Pediculus humanus capitis  UMLS: 2168807

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000743
- label: Pediculosis capitis
- parents: [Positive skin infectious agent test (HP:0430117)](https://hpo.jax.org/app/browse/term/HP:0430117)
- pmid: PMID:29262055
- def: Visualization of at least one louse on visual inspection of the head hair. Use of a bright light, magnifying lens, or fine-toothed comb (lice comb) can aid in diagnosis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7999
- synonyms:
